components/availability/src/Availability.svelte

```diff
@@ -335,7 +354,11 @@
         if (
           numFreeCalendars === 0 ||
           (numFreeCalendars !== allCalendars.length &&
-            numFreeCalendars < allCalendars.length * partial_bookable_ratio)
+            numFreeCalendars < allCalendars.length * partial_bookable_ratio) ||
+          (required_participants.length &&
+            !required_participants.every((participant) =>
+              epoch[0].available_calendars.includes(participant),
```

Comment:
Same here re: making this into a variable so it's more readable, e.g. `allParticipantsUnavailable = !required_participants.every((participant) => epoch[0].available_calendars.includes(participant))`